rel/server/env.sh.eex

```diff
@@ -3,8 +3,7 @@ export RELEASE_DISTRIBUTION=${LIVEBOOK_DISTRIBUTION:-${RELEASE_DISTRIBUTION:-sna
 export RELEASE_MODE=interactive
 
 cookie_path="${RELEASE_ROOT}/releases/COOKIE"
-if [ ! -f $cookie_path ]; then
+if [ ! -f $cookie_path ] && [ -z "$RELEASE_COOKIE" ]; then
   RELEASE_COOKIE=$(cat /dev/urandom | env LC_ALL=C tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
-  echo "$RELEASE_COOKIE" > "$cookie_path"
-  [ $? -eq 0 ] || export RELEASE_COOKIE
+  echo "$RELEASE_COOKIE" > "$cookie_path" || export RELEASE_COOKIE
```

Comment:
@josevalim I reverted 497f372140dde08e74bc1b15c39a3cd9fdc5b98f, since now we will only export the random cookie if we can't write **and** no cookie is set. So if someone is concerned with random cookie on read-only fs they can just set the cookie themselves.